tests/resources/bibtex/1_out.json

```diff
@@ -0,0 +1,67 @@
+{
+  "abstract": "... to 100 {\\%}(concurrent intercalation)...",
+  "author": "Shi, % some comments here\nFazhan and Rong, Xing and Xu, Nanyang and Wang, Ya and Wu, Jie and Chong, Bo and Peng, Xinhua and Kniepert, Juliane and Schoenfeld, Rolf-Simon and Harneit, Wolfgang and Feng, Mang and Du, Jiangfeng",
```

Comment:
This seems to be a problem with bibtexparser. It looks like it's not great at parsing comments (see [this issue](https://github.com/sciunto-org/python-bibtexparser/issues/133))